src/capi_frontend/capi.cpp

```diff
@@ -336,11 +335,14 @@ OVMS_Status* OVMS_ServerMultiModelOptionsSetConfigPath(OVMS_ServerMultiModelOpti
     return nullptr;
 }
 // inference API
-OVMS_Status* OVMS_InferenceRequestNew(OVMS_InferenceRequest** request, const char* servableName, uint32_t servableVersion) {
+OVMS_Status* OVMS_InferenceRequestNew(OVMS_InferenceRequest** request, OVMS_Server* server, const char* servableName, uint32_t servableVersion) {
```

Comment:
why is it called New?

Reply:
Because it allocates new object InferenceRequest with new operator. It should correspond with normal new/delete operators for c++. It also the same convention as Triton C-API has.